odo/core.py

```diff
@@ -1,12 +1,15 @@
 from __future__ import absolute_import, division, print_function
 
+import logging
 from contextlib import contextmanager
 from warnings import warn
 
 import networkx as nx
 from datashape import discover
 from .utils import expand_tuples, ignoring
 
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
```

Comment:
I think I misunderstood initially what NullHandler was for.  It's not needed here and I will remove it.